torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -70,7 +87,18 @@ def gaussian_blur_image_tensor(
     else:
         needs_unsquash = False
 
-    output = _FT.gaussian_blur(image, kernel_size, sigma)
+    dtype = image.dtype if torch.is_floating_point(image) else torch.float32
+    kernel = _get_gaussian_kernel2d(kernel_size, sigma, dtype=dtype, device=image.device)
+    kernel = kernel.expand(image.shape[-3], 1, kernel.shape[0], kernel.shape[1])
+
+    image, need_cast, need_squeeze, out_dtype = _FT._cast_squeeze_in(image, [kernel.dtype])
+
+    # padding = (left, right, top, bottom)
+    padding = [kernel_size[0] // 2, kernel_size[0] // 2, kernel_size[1] // 2, kernel_size[1] // 2]
+    output = _FT.torch_pad(image, padding, mode="reflect")
+    output = _FT.conv2d(output, kernel, groups=output.shape[-3])
```

Comment:
These two are just imported from `torch`, right? If so, can we also do that here instead importing from the legacy transforms? 

Plus, regarding the `conv2d`, could we maybe implement a common `nd_conv2d` as proposed in https://github.com/pytorch/vision/pull/6726#issue-1402722696? Basically, `nd_conv2d` should handle the `_cast_squeeze_in` and `_cast_squeeze_out` so the actual kernel is cleaner. That way, all kernels that use `conv2d` now could use that utility and don't need to worry about shape or dtype.

Reply:
OK, I can import methods directly. 

As for `nd_conv(2d?)`, I do not think we always have a situation when we can hide _cast_squeeze_in, _cast_squeeze_out inside `nd_*` op... Anyway, let's see that elsewhere (not this PR).